src/modules/relewise/service.ts

```diff
@@ -0,0 +1,74 @@
+import { ProductDTO } from "@medusajs/framework/types";
+import { Integrator, ProductAdministrativeActionBuilder, ProductUpdateBuilder } from "@relewise/integrations"
+import { DataValueFactory, Trackable } from "@relewise/client"
+
+type RelewiseOptions = {
+  datasetId: string;
+  apiKey: string;
+  serverUrl: string;
+  language: string | null;
+}
+
+class RelewiseService {
+  private integrator: Integrator;
+  private options: RelewiseOptions;
+
+  constructor({}, options: RelewiseOptions) {
+    this.integrator = new Integrator(options.datasetId, options.apiKey, { serverUrl: options.serverUrl })
+    this.options = options;
+  }
+
+  async Sync(products: ProductDTO[]) {
+    const date: number = Date.now();
+
+    const productUpdates: Trackable[] = [];
+    
+    products.forEach(product => {
+      const productUpdate = new ProductUpdateBuilder({
+        id: product.id,
+        productUpdateKind: 'ReplaceProvidedProperties',
+      })
+      .displayName([{ language: this.options.language ?? "en", value: product.title }])
+      .data({ 'ImportedAt': DataValueFactory.number(date) });
+
+      productUpdates.push(productUpdate.build());
+    });
+
+    const disableNonUpdatedProducts = new ProductAdministrativeActionBuilder({
+      language: null,
+      currency: null,
+      filters(filterBuilder) {
+          filterBuilder
+              .addProductDataFilter(
+                  'ImportedAt',
+                  (conditionBuilder) => conditionBuilder.addEqualsCondition(
+                      DataValueFactory.number(date), true),
+                  true,
+              );

```

Comment:
```suggestion
              .addProductDataFilter('ImportedAt', (conditionBuilder) => conditionBuilder.addEqualsCondition(DataValueFactory.number(date), true));
```

Also the last `true` is not needed as it is default

Reply:
Check